Plugins/GeoModel/include/Acts/Plugins/GeoModel/GeoModelDetectorElement.hpp

```diff
@@ -15,7 +15,10 @@
 
 #include <memory>
 
+#include <GeoModelKernel/GeoFullPhysVol.h>
+
 class GeoFullPhysVol;
```

Comment:
I guess this forward declare can be dropped after the include?

Reply:
You could also just rely on the GeoVFullPhysVol

Reply:
You are right!